lib/project_types/extension/features/argo_serve.rb

```diff
@@ -24,13 +24,55 @@ def specification
         specification_handler.specification
       end
 
+      def tunnel_required?
+        !!flags[:tunnel]
+      end
+
+      def tunnel
```

Comment:
I can't find where this method is being invoked. I'm puzzled, maybe its too late.

Reply:
This was meant to be [invoked here](https://github.com/Shopify/shopify-app-cli/pull/1191/files#diff-4b2e338f85e70c84e53e134866f50041a662b929bcc5762b05d9f5f42260eab3R55), I was just using the wrong name. Good catch! :)